client/my-sites/importer/style.scss

```diff
@@ -117,7 +117,9 @@
 
 .importer__upload-progress,
 .importer__import-progress {
-	width: 80%;
```

Comment:
Why did this change?

Reply:
A change in the application CSS - possibly in the order of stylesheets - meant that 80% was being overridden with a 100%.